tests/filename/filenametest.cpp

```diff
@@ -710,8 +712,24 @@ TEST_CASE("wxFileName::Exists", "[filename]")
     // These files are only guaranteed to exist under Linux.
     // No need for wxFILE_EXISTS_NO_FOLLOW here; wxFILE_EXISTS_SYMLINK implies it
     CHECK( wxFileName::Exists("/proc/self", wxFILE_EXISTS_SYMLINK) );
-    CHECK( wxFileName::Exists("/dev/log", wxFILE_EXISTS_SOCKET) );
 #endif // __LINUX__
+    wxString name = dirTemp.GetPath() + "/socktmpdirXXXXXX";
+    wxString socktempdir = wxString::From8BitData(mkdtemp(name.char_str()));
+    wxString sock = socktempdir + "/socket";
+    struct sockaddr_un addr;
+    int fd = socket(AF_UNIX, SOCK_STREAM, 0);
```

Comment:
I wonder if we could use the Unix socket created by `wxTCPServer` instead of redoing all this manually?

Reply:
Yep.  I swear I grepped to see if there was existing code that did Unix sockets... :P